src/workers/parser/getSymbols.js

```diff
@@ -235,6 +248,11 @@ export default function getSymbols(source: Source): SymbolDeclarations {
   return symbols;
 }
 
+export function getHasJSX(source: Source): boolean {
+  const symbols = getSymbols(source);
+  return symbols.hasJSX.length > 0;
```

Comment:
we should store this property in `sourceMetaData` in `reducers/ast` along side the `isReact` field. This way we don't need to ask the worker every time